code/modules/vehicles/scooter.dm

```diff
@@ -74,6 +75,25 @@
 		visible_message("<span class='danger'>[src] crashes into [A], sending [H] flying!</span>")
 		playsound(src, 'sound/effects/bang.ogg', 50, 1)
 
+/obj/vehicle/scooter/skateboard/MouseDrop(atom/over_object)
+	var/mob/living/carbon/M = usr
+	if(!istype(M) || M.incapacitated() || !Adjacent(M))
+		return
+	if(has_buckled_mobs() && over_object == M)
+		M << "<span class='warning'>You can't lift this up when somebody's on it.</span>"
+		return
+	if(over_object == M)
+		if(!linked_board)
+			linked_board = new /obj/item/weapon/melee/skateboard(src)
+			linked_board.linked_board = src
+		forceMove(linked_board)
```

Comment:
remind me why it's bad to delete and recreate objects and have a stable PR again